src/input.tsx

```diff
@@ -168,7 +168,7 @@ export default class Input extends React.PureComponent<IProps, unknown> {
         )}
         <input
           className={classes}
-          id={`geosuggest__input${this.props.id ? `--${this.props.id}` : ''}`}
+          id={this.props.id ? this.props.id : 'geosuggest__input'}
```

Comment:
I would prefer to change the `htmlFor` according to the input id instead to more likely not compete with other ids. (Shouldn't make a difference in most cases, but feels a little better to me.)

Reply:
Hello @plumdumpling not quite sure what the suggestion is here, to adapt the label instead? Like such ? : 

```
{shouldRenderLabel && (
          <label
            className="geosuggest__label"
            htmlFor={this.props.id ? this.props.id : 'geosuggest__input'}>
            {this.props.label}
          </label>
        )}
        <input
          className={classes}
          id={`geosuggest__input${this.props.id ? `--${this.props.id}` : ''}`}
```

Reply:
would you rather keep the existing `geosuggest__input${this.props.id ? `--${this.props.id}` : ''}` ? 

let me know what you think of the changes made.

Reply:
Yes I would keep the existing input id and adjust the label's `htmlFor` accordingly. So it would be ...
```jsx
<label
  className="geosuggest__label"
  htmlFor={`geosuggest__input${this.props.id ? `--${this.props.id}` : ''}`}>
  {this.props.label}
</label>
```
... for the label and ...
```jsx
<input
  className={classes}
  id={`geosuggest__input${this.props.id ? `--${this.props.id}` : ''}`}
  ...
/>
```
... for the input.

Reply:
Thanks adapted this now